### DISCUSSION TITLE
What with MySql database

### DISCUSSION THREAD

**Discussion by wcizmowski** [dunglas/symfony-docker#345] [General]
Hi,
What is the way to use mysql database in this project/docker ?
I need esomething external ?

**Comment by maxhelias** [comment]
Hi
See the documentation about the extra service with Flex : https://github.com/dunglas/symfony-docker/blob/main/docs/extra-services.md or  see the command "make:docker:database" on the maker bundle to choose between MySQL, MariaDB or Postgres ([symfony/maker-bundle@main/src/Maker/MakeDockerDatabase.php](https://github.com/symfony/maker-bundle/blob/main/src/Maker/MakeDockerDatabase.php)).
